.github/workflows/cypress.yml

```diff
@@ -50,6 +50,8 @@ jobs:
         run: ./vegawallet import -w UI_Trading_Test --recovery-phrase-file ./recovery -p ./passphrase
       - name: Import config
         run: ./vegawallet network import --from-url https://raw.githubusercontent.com/vegaprotocol/networks/master/fairground/fairground.toml
+      - name: Create new public key
+        run: ./vegawallet key generate -w UI_Trading_Test -p ./passphrase
```

Comment:
minot (non-blocking): I'd just call this key 2 as it i always just the second key

Reply:
That's renamed now